ios/Classes/SwiftReceiveSharingIntentPlugin.swift

```diff
@@ -149,7 +149,8 @@ public class SwiftReceiveSharingIntentPlugin: NSObject, FlutterPlugin, FlutterSt
                         guard let path = getAbsolutePath(for: $0.path) else {
                             return nil
                         }
-                        return SharedMediaFile.init(path: $0.path, thumbnail: nil, duration: nil, type: $0.type)
+                        let pathWithoutPrefix = String($0.path.dropFirst(7))//knock 'file://' off here...
```

Comment:
Why this change?

Reply:
This patch is taken from here https://github.com/KasemJaffer/receive_sharing_intent/issues/141. @LazyDave76 is the author of both patches.

Reply:
The shared media object has a path property which already contains "file://" on iOS.  

This results in file not found when looking up the path in iOS because the value looks like "file://file://PathToCopiedFile"

Reply:
having 'fixed' this by in a pretty hacky way - this is probably what it should look like:
```suggestion
                        return SharedMediaFile.init(path: path, thumbnail: nil, duration: nil, type: $0.type)
```

seems that the path is url encoded though and there are fun times with files with spaces...